backend/inspirehep/migrator/tasks.py

```diff
@@ -344,11 +344,16 @@ def migrate_record_from_mirror(
     logger = LOGGER.bind(recid=prod_record.recid)
     try:
         json_record = marcxml2record(prod_record.marcxml)
+    except NotSupportedError as exc:
+        logger.warning(str(exc), recid=prod_record.recid)
+        prod_record.valid = True
+        db.session.merge(prod_record)
+        return
```

Comment:
`return None` as per PEP8 (the statement is not used only for control-flow, this is a function that actually returns a value, so all branches should return a value).